playbooks/restore.yml

```diff
@@ -0,0 +1,19 @@
+---
+#
+# valet.sh | restore
+#
+# @author: "Johann Zelger"
+# @author: "Philipp Dittert"
+# @command: "restore"
+# @description: "restore persistent instance resources (eg. db, media-files) | Usage: valet.sh restore test <previous|latest>"
+# @usage: "valet.sh restore <instancebackup-identifier> <valet_restore_db_identifier>
+
```

Comment:
please add @help comment line for details command help output when calling:
`valet.sh -h restore`